src/main/java/nl/tudelft/contextproject/model/level/util/CorridorType.java

```diff
@@ -0,0 +1,155 @@
+package nl.tudelft.contextproject.model.level.util;
+
+import nl.tudelft.contextproject.model.level.MazeTile;
+import nl.tudelft.contextproject.model.level.TileType;
+
+/**
+ * Enum for different kinds of corridor types.
+ * Used in CorridorBeautification.
+ * <P>
+ * In this class, the java doc each enum represents a certain configuration of a corridor.
+ * Each corridor is drawn, with A being the current corridor tile and C the corridor tiles around it.
+ * The horizontal corridors and vertical corridors are just as shown, but the corner corridors are defined
+ * from the direction where the extension will take place, as it was easier to keep track of the position that way.
+ * 
+ */
+public enum CorridorType {
+
+	/**
+	 * A Vertical corridor.
+	 * <p>
+	 * &ensp;C&ensp;<br>
+	 * &ensp;A&ensp;<br>
+	 * &ensp;C&ensp;<br>
+	 */
+	VERTICAL,
+
+	/**
+	 * An Horizontal corridor.
+	 * <p>
+	 * &ensp;&ensp;&ensp;<br>
+	 * CAC<br>
+	 * &ensp;&ensp;&ensp;<br>
+	 * <br>
+	 */
+	HORIZONTAL,
+
+	/**
+	 * An Upper Right corridor.
+	 * <p>
+	 * &ensp;&ensp;&ensp;<br>
+	 * CA&ensp;<br>
+	 * &ensp;C&ensp;<br>
+	 * <br>
+	 */
+	UPPER_RIGHT,
+
+	/**
+	 * An Upper Left corridor.
+	 * <p>
+	 * &ensp;&ensp;&ensp;<br>
+	 * &ensp;AC<br>
+	 * &ensp;C&ensp;<br>
+	 * <br>
+	 */
+	UPPER_LEFT,
+
+	/**
+	 * A Lower Right corridor.
+	 * <p>
+	 * &ensp;C&ensp;<br>
+	 * CA&ensp;<br>
+	 * &ensp;&ensp;&ensp;<br>
+	 * <br>
+	 */
+	LOWER_RIGHT,
+
+	/**
+	 * A Lower Left corridor.
+	 * <p>
+	 * &ensp;C&ensp;<br>
+	 * &ensp;AC<br>
+	 * &ensp;&ensp;&ensp;<br>
+	 * <br>
+	 */
+	LOWER_LEFT,
+
+	/**
+	 * A Not defined corridor.
+	 * <p>
+	 * ???<br>
+	 * ?A?<br>
+	 * ???<br>
+	 * <br>
+	 */
+	NOT_DEFINED;
+
+	/**
+	 * Get CorridorType from Map.
+	 * Scan for each possible corridor tile around the x, y corridor tile.
+	 * 
+	 * @param map
+	 * 		map to check
+	 * @param x
```

Comment:
X, y, z, i  and j are all acceptable one letter variable, parameter names as they all are descriptive of their function. This was discussed during the meeting with the TA's two weeks ago.